src/rpc/blockchain.cpp

```diff
@@ -808,6 +808,30 @@ CoinStatsHashType ParseHashType(const std::string& hash_type_input)
     }
 }
 
+/**
+ * Calculate statistics about the unspent transaction output set
+ *
+ * @param[in] index_requested Signals if the coinstatsindex should be used (when available).
+ */
+std::optional<kernel::CCoinsStats> GetUTXOStats(CCoinsView* view, node::BlockManager& blockman,
```

Comment:
nit: could be made static, since this function is not used outside this module
```suggestion
static std::optional<kernel::CCoinsStats> GetUTXOStats(CCoinsView* view, node::BlockManager& blockman,
```

Reply:
Fixed in e6e2b165fd377806e257fbb5378e22067a9a3957, thanks!